### ISSUE TITLE
Details about Evaluation on SKEMPI v2

### ISSUE THREAD

**Issue by mylRalph** [EureKaZhu/DiffAffinity#3]
Hi, congrats on your excellent work! I am confused about several details regarding the evaluation of DiffAffinity on SKEMPI v2.

1. **Splitting Strategy**: In your published paper, I noticed that three-fold cross-validation was conducted to benchmark the model, where two of the folds were utilized for training and validation, with the remaining fold reserved for testing purposes. However, when I checked the codebase, I found there was no reserved fold for testing. In fact, two folds were used for training, and the remaining one was for validation. I wonder if the test set is exactly the same as the validation set.
2. **Selection of Checkpoints**: Unlike RDE-Net, all the metrics of 3 DiffAffinity models were calculated independently. I guess the best checkpoint was selected individually for each of the 3 DiffAffinity models. However, in RDE-Net, metrics in each validation step were obtained from all the predicted △△G of 3 models together, and the checkpoint was selected for 3 models as a whole, which meaned the selected checkpoint might not be the best on some folds. These two ways of checkpoint selection might lead to differences when reporting the results. How do you select the checkpoints during your evaluation of DiffAffinity on SKEMPI? Do you use the best per-structure pearson for selection?

I just want to follow the same experimental setting as DiffAffinity so I can make a fair comparison with your results. I would appreciate it if I could get your help! Thank you!

**Comment by TianZhuAI4S** [comment]
1. The test set serves as the validation set.
2. We employ checkpoints from all three folds to predict outcomes for each test set, subsequently averaging these results instead of selecting the best one.

**Comment by mylRalph** [comment]
Hi, thanks for your quick reply! I understand that results of 3 models are averaged in the end. I mean each of the 3 checkpoints is the best one for each of the 3 DiffAffinity models. By the way, do you use the best per-structure pearson as the selecting criterion?

**Comment by TianZhuAI4S** [comment]
No, we just set the same seed.
